src/test/rustdoc/issue-80893.rs

```diff
@@ -0,0 +1,6 @@
+// compile-flags: --test -Z unstable-options --test-builder true
```

Comment:
rustdoc passes the source code the the builder through stdin. If `true` happens to terminate before that, the rustdoc will be terminated by SIGPIPE. This makes the test quite flaky, e.g., https://github.com/rust-lang/rust/pull/79705#issuecomment-762919291

Reply:
Could we use `cat` instead ?

Reply:
Or maybe `wc` if the output would end up in the test log.

Reply:
cat: unrecognized option '--crate-type'

Reply:
Hmm, in shell `:` will work, but I think this goes through `process::Command`.

Reply:
You can always just use `--test-builder rustc`.

Reply:
> Hmm, in shell `:` will work, but I think this goes through `process::Command`.

No, `:`, if it works, is just an alias for `true`.  It would have the same problem.  We need something that will actually read the input.

> You can always just use `--test-builder rustc`.

Haha lol, why didn't I think of that.  Of course that wouldn't test that the argument actually has any effect but the current test doesn't check that anyway.  So +1.

Reply:
I opened https://github.com/rust-lang/rust/pull/81197 fixing this.